File: pset1/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height; j++)
        {
            if (j + 1 < height - i)
            {
                putchar(' ');
            }
            else
            {
                putchar('#');
            }
        }
        putchar('\n');
    }

    return 0;
}

File: pset6/mario-less/mario.py
from cs50 import get_int

# Input pyramid height (1 ~ 8)
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

# Create pyramid
for h in range(height):
    print(" " * (height - h - 1) + "#" * (h + 1))
